tasks/fh-build.js

```diff
@@ -39,6 +40,31 @@ module.exports = function(grunt) {
   }
 
   /**
+   * Keep just the first occurence of `node_modules` and the directory name next to it, and append `**`
+   * e.g. /home/user/work/fh-ngui/node_modules/watchify/node_modules/assert will be formatted  to node_modules/watchify/**
+   */
+  var formatDirectory = function(dir) {
```

Comment:
This function works fine in the one place it's currently used, because we've already filtered out anything that doesn't have a 'node_modules' part. However, as a standalone function like this, it's possible that someone will want to use it elsewhere here in future, and won't know that they can't give it input that doesn't have a 'node_modules' part (otherwise a vague  TypeError is thrown)